sdk/maps/Azure.Maps.Geolocation/tests/Samples/GeolocationClientSamples.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+#region Snippet:GeolocationImportNamespace
+using Azure.Maps.Geolocation;
+#endregion
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Maps.Geolocation.Tests
+{
+    public class GeolocationClientSamples : SamplesBase<GeolocationClientTestEnvironment>
+    {
+        public void GeolocationClientViaAAD()
+        {
+            #region Snippet:InstantiateGeolocationClientViaAAD
+            // Create a MapsGeolocationClient that will authenticate through Active Directory
+            var credential = new DefaultAzureCredential();
+            var clientId = "<My Map Account Client Id>";
+            MapsGeolocationClient client = new MapsGeolocationClient(credential, clientId);
+            #endregion
+        }
+
+        public void GeolocationClientViaSubscriptionKey()
+        {
+            #region Snippet:InstantiateGeolocationClientViaSubscriptionKey
+            // Create a MapsGeolocationClient that will authenticate through Subscription Key (Shared key)
+            var credential = new AzureKeyCredential("<My Subscription Key>");
+            MapsGeolocationClient client = new MapsGeolocationClient(credential);
+            #endregion
+        }
+
+        [Test]
+        public void GetLocationTest()
+        {
+            var credential = new DefaultAzureCredential();
+            var clientId = TestEnvironment.MapAccountClientId;
+            var client = new MapsGeolocationClient(credential, clientId);
+
+            #region Snippet:GetLocation
+            //Get location by given IP address
+            var ipAddress = "2001:4898:80e8:b::189";
+            var result = client.GetLocation(ipAddress);
```

Comment:
Based on comments from Route and Render: We generally discourage using `var` in sample code when not doing something explicitly obvious like creating an instance. **We should use the type names across all samples to ensure readability for developers unfamiliar with the Maps API.**
```suggestion
            string result = client.GetLocation(ipAddress);
```